src/index.js

```diff
@@ -187,6 +187,8 @@ class Node extends EventEmitter {
           return each(this.modules.discovery, (d, cb) => d.start(cb), cb)
         }
         cb()
+
+        this.emit('online')
```

Comment:
Agree, "online" is somewhat ambiguous, and not the goal. I'd say started and stopped are what are the really important events for the user (at the libp2p and ifs levels).

Following that trail, I think this leads to replacing `.isOnline()`  with `.isStarted()`, no?